### PR TITLE
Add Carousel component for embedded video

### LINKED ISSUE #15: Find out how to embed YouTube video galleries
Several EIC content pages include reels of videos (hosted on YouTube) like this:

image

How do we present these in the first round? To me, this feature has low priority for the first release and should consider shortcuts like

1. A playlist embed snippet from YouTube in an iframe
2. A grid / reel of cards that are external links to the YouTube videos
3. Something else simple that does not require new components

## Acceptance criteria

- [ ] Found out how to embed YouTube video galleries for first release

### PR DESCRIPTION
**Pull request by hanbyul-here**
- It is good to keep in mind that users rarely interact with carousels. When vertical space is allowed, it is considered better not to use it: https://www.smashingmagazine.com/2022/04/designing-better-carousel-ux/#do-users-actually-use-carousels 
- The carousel in this pr is not meant for any element, it is specifically for embedded video.

Close #15 
example video carousel:  https://deploy-preview-23--earth-information-center.netlify.app/stories/hyperwall

### CODE REVIEW

File: overrides/common/embedded-video-carousel.tsx [hunk 1]
```diff
@@ -117,49 +117,58 @@ export default function Carousel({ items }: EmbeddedVideosPropType) {
...
+            <Slider>
+              {items.map((t, idx) => (
+                <FeaturedContent
+                  key={t.title}
+                  role="group"
+                  aria-roledescription="Slide"
+                  aria-labelledby={`carousel-item-${idx}__label`}
+                >
+                  <Slide index={idx}>
+                    <Embed height={height} src={t.src}></Embed>
```
Comment (sandrahoang686) [inline overrides/common/embedded-video-carousel.tsx]: `<Embed height={height} src={t.src}/>` ?

**Review by sandrahoang686 — APPROVED** [review]

File: stories/locfeature.HYPER.mdx [hunk 2]
```diff
@@ -25,8 +25,12 @@ taxonomy:
     values:
       - Hyperwall
 ---
-
-import Carousel from './components/carousel.jsx';
+import Carousel from "../overrides/common/embedded-video-carousel";
```
Comment (hanbyul-here) [inline stories/locfeature.HYPER.mdx:28]: @agurvich I hope how to use this carousel is clear to you. I used the `carousel_content.json` to feed the data to carousel.

**Comment by agurvich** [comment]
Wow!!!!! @hanbyul-here this is amazing, I can't believe you got this working so fast. It looks so good. The only feedback I have is that in my latest version of the carousel I have a counter for the index (something like `index+1/contentArray.length`) so that you can tell how far you are into the carousel.

**Comment by hanbyul-here** [comment]
@agurvich I that might require some changes on carousel component. I recommend using this component for your carousel need for now, and I will see what I can do with the rest of the time.

**Comment by agurvich** [comment]
Gotcha, I'll go ahead and merge these commits into my working branch and get that all transferred over.

**Comment by hanbyul-here** [comment]
@agurvich I will merge this branch to `develop` so we can have the same base.

**Comment by agurvich** [comment]
Thanks @hanbyul-here! I just opened #29  for integrating your carousel into the rest of the stories. I'll fiddle with the rest of the content and merge later today. I will do a separate PR for the landing page video, content, and combing over the pages for references to GHG & any placeholder text.

**Pull request merged**